bedrock/privacy/templates/privacy/notices/firefox-cliqz-quantum.html

```diff
@@ -0,0 +1,72 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "base-pebbles.html" %}
+
+{% block page_title %}
+  {{ doc.body.section.h2.get_text() }}
+{% endblock %}
+
+{% block page_css %}
+  {% stylesheet 'firefox_privacy' %}
+{% endblock %}
+
+{% block body_id %}firefox-notice{% endblock %}
+
+{% block string_data %}
```

Comment:
Rather than add these explicitly for every new template, it may be a better to make a common base tempalte from which all redesigned privacy notices can inherit from? You could apply the same to the CSS/JS bundles which are the same for each page.

Reply:
Yup, as I opened this I was thinking it would be worth having a new base at this point that this one and the previous one can extend. Will do that.